packages/shared/src/components/profile/SocialChips.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ReactElement } from 'react';
+import GitHubIcon from '../icons/GitHub';
+import TwitterIcon from '../icons/Twitter';
+import LinkIcon from '../icons/Link';
+import { Button, ButtonSize } from '../buttons/Button';
+
+export interface SocialChipsProps {
+  links: { github?: string; twitter?: string; portfolio?: string };
+}
+
+const handlers: Record<
+  keyof SocialChipsProps['links'],
+  {
+    icon: ReactElement;
+    href: (x: string) => string;
+    label: (x: string) => string;
+  }
+> = {
+  github: {
+    icon: <GitHubIcon />,
+    href: (x) => `https://github.com/${x}`,
+    label: (x) => `@${x}`,
+  },
+  twitter: {
+    icon: <TwitterIcon />,
+    href: (x) => `https://twitter.com/${x}`,
+    label: (x) => `@${x}`,
+  },
+  portfolio: {
+    icon: <LinkIcon />,
+    href: (x) => x,
+    // Strip protocol from url
+    label: (x) => x.replace(/(^\w+:|^)\/\//, ''),
+  },
+};
+const order: (keyof SocialChipsProps['links'])[] = [
+  'github',
+  'twitter',
+  'portfolio',
+];
+
+export function SocialChips({ links }: SocialChipsProps): ReactElement {
+  const elements = order
+    .filter((key) => !!links[key])
+    .map((key) => (
+      <Button
```

Comment:
Did you choose the old button v1 on purpose?
Technically it should be unused soon as we plan to remove it.

Reply:
Yes I copied an existing implementation that Tsahi referred me to. We'll be able to migrate this button only when we introduce the "Subtle" variation. Don't know the timeline for that though

Reply:
Ah ok got it :) 
Yeah there are 3/4 exclusions now we need to find out still so no problem for this PR.